docker/override-network.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+set -e
+
```

Comment:
what about using `envsubst` looks much cleaner approach to me (the article says it's already in the nginx image, didn't checked) https://ledermann.dev/blog/2018/04/27/dockerize-and-configure-javascript-single-page-application/

Reply:
I'm not sure about workflow using it. Currently, I have index.html that works with and without Docker. With `envsubst` I need a template version of index.html to use in Docker, and without docker I would need something to remove the template part. Alternatively, I can maintain index.html and index.template.html with almost the same content 🤷‍♀️
Or do it in index.html like
```js
<script>
window.overrideNetwork = {
  name: '$VUE_APP_NETWORK_NAME',
  url: '$VUE_APP_NODE_URL',
  middlewareUrl: '$VUE_APP_MIDDLEWARE_URL',
  explorerUrl: '$VUE_APP_EXPLORER_URL',
  compilerUrl: '$VUE_APP_COMPILER_URL',
};
if (window.overrideNetwork.name.startsWith('$')) {
  delete window.overrideNetwork;
}
</script>
```
?

Reply:
That's actually not bad idea - to drop the config template if not replaced. Outside docker, well this is why I asked above how do I do it in local dev environment. There was functionality for that which you dropped (which the process environment).